src/com/lftechnology/java/training/dipak/multiples/Multiples.java

```diff
@@ -0,0 +1,95 @@
+package com.lftechnology.java.training.dipak.multiples;
+
+/*	
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+ *Find the sum of all the multiples of 3 or 5 below 1000.
+*/
+
+import java.util.Scanner;
+
+import java.util.logging.Logger;
+
+public class Multiples {
+	private final static Logger LOGGER = Logger.getLogger(Multiples.class.getName());
+	int multiple[];	
+	int index=0;
+	int range;		
+	Multiples(int a){
+		range=a;
+		multiple=new int[range];	
+	}
+	
+	/**
+	 * <p>This method check all the multiples of the user defined numbers that are taken as input within the user
+	 * defined range </p>
+	 * @param  array a[]
+	 * @return Doesn't return a value
+	 * @author <dipakthapa@lftechnology.com>
+	 */
+	
+	void getMultiples(int a[]){
+		for(int i=0;i<a.length;i++){
+			for(int j=0;j<range;j++){		//Change according to the range
+				if((j % a[i]) == 0 ){
+					 boolean exists=ifexists(j);
+					if(!exists){
+						multiple[index++]=j;
+					}
+				}
+			}
+		}		
+	}
+	/**
+	 * <p>This method checks whether the given number is a multiple of the user defined numbers or not. It returns a boolean value </p>
+	 * @param  a
+	 * @return temp
+	 * @author <dipakthapa@lftechnology.com>
+	 */
+	boolean ifexists(int a){
+		boolean temp=false;
+		for(int k=0;k<index;k++){
+			if(a == multiple[k]){
+				temp=true;

```

Comment:
once the match is found.Is it required to continue the loop.better to break the loop using break;